app/views/helpers/injected/AddressHelper.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import models.core.AddressModel
+
+@this()
+@(address: AddressModel)(implicit request: Request[_], messages: Messages)
```

Comment:
`request` is not used.
```suggestion
@(address: AddressModel)(implicit messages: Messages)
```